code/modules/projectiles/magazines/smgs.dm

```diff
@@ -64,6 +64,7 @@
 	name = "\improper M39 HV extended magazine (10x20mm)"
 	desc = "A 10x20mm caseless HV extended submachinegun magazine. Powerful propellant allows the bullet increased travel speed and minor penetration capabilities, noticeably improving its efficacy at long ranges, although it still suffers significantly compared to a rifle bullet."
 	max_rounds = 72
+	w_class = SIZE_MEDIUM // big ammo big size DUHH
```

Comment:
add this to other extended smg mags if they have a significantly bigger magsize too

Reply:
already

Reply:
someone planned ahead of me and all big mags were already MEDIUM size with defines on each one